CHANGES.rst

```diff
@@ -28,6 +28,9 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Support for previously deprecated ``clobber`` keyword has been removed. Use
```

Comment:
Usually we mention the deprecations and removal of previous deprecations in the API change section below.